src/js/request_scheduler.js

```diff
@@ -192,8 +201,8 @@ class RequestScheduler {
                     alert('Amazon Order History Reporter Chrome Extension\n\n' +
                           'It looks like you might have been logged out of Amazon.\n' +
                           'Sometimes this can be "partial" - some types of order info stay logged in and some do not.\n' +
-                          'I will now attempt to open a new tab with a login prompt. Please use it to login,\n' +
-                          'and then retry your chosen orange button.');
+                          'I will now attempt to open a new tab with a login prompt.\n' +
+                          'Please use it to login, and then return here and retry your chosen date button.');
```

Comment:
I reckon people are getting really fed up with this. I'm wondering if we should ask users to provide their credentials and store them in a cookie (locally) and use it on their behalf. I intend to ask some browser security experts I know about any risks this might introduce (I'm starting back at Google next month, so it'll be easy to get face-to-face time with them).
What do you think?